slyd/slyd/resources/spiders.py

```diff
@@ -56,7 +56,7 @@ def update_spider(manager, spider_id, attributes):
         if 'samples' in spider:
             del spider['samples']
         response = SpiderSchema(context=context).dump(spider).data
-        spider = _populate_relationships(spider, spider_id)
+        spider = _populate_relationships(manager, spider, spider_id)
```

Comment:
You can pass `manager.project_name` to `_populate_relationships` as only the `project_name` is used in `_populate_relationships`.